src/containers/Alerts/index.js

```diff
@@ -49,7 +50,7 @@ function AlertsComponent(props) {
 						className="shadow alerts-style"
 						fade={true}
 						isOpen={!alert.acked}
-						toggle={() => store.dispatch({ type: ACK_ALERT, key: alert.key })}
+						toggle={!sessionExpiration ? () => store.dispatch({ type: ACK_ALERT, key: alert.key }) : null}
```

Comment:
@Aringoaway PLease add a comment that your change will remove the close button for alert with expiration